<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BathyViewer</class>
 <widget class="QWidget" name="BathyViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BathyView</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>781</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SavenewlvlPushButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Data Viewer</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="AdjustBathydoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>271</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>240</y>
     <width>223</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adjust reference LVL [m]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>811</x>
     <y>61</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select H Value</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="hGroupBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>100</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Select sign of H below the surface</string>
   </property>
   <widget class="QRadioButton" name="greaterRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>59</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>H &lt; 0</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lesserRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>59</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>H &gt; 0</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
